src/manager/node.c

```diff
@@ -61,7 +62,8 @@ static const sd_bus_vtable node_vtable[] = {
         SD_BUS_METHOD("SetUnitProperties", "sba(sv)", "", node_method_set_unit_properties, 0),
         SD_BUS_METHOD("EnableUnitFiles", "asbb", "ba(sss)", node_method_passthrough_to_agent, 0),
         SD_BUS_METHOD("DisableUnitFiles", "asb", "a(sss)", node_method_passthrough_to_agent, 0),
-        SD_BUS_METHOD("Reload", "", "", node_method_passthrough_to_agent, 0),
+        SD_BUS_METHOD("SetLogLevel", "s", "", node_method_set_log_level, 0),
+        SD_BUS_METHOD("Reload", "", "", NULL, 0),
```

Comment:
```suggestion
        SD_BUS_METHOD("Reload", "", "", node_method_passthrough_to_agent, 0),
```
This seems wrong. Please re-add the method as it was before.